Modules/IndividualAssessment/classes/class.ilObjIndividualAssessment.php

```diff
@@ -66,9 +66,12 @@ public function __construct($a_id = 0, $a_call_by_reference = true)
     /**
      * @inheritdoc
      */
-    public function create()
+    public function create($a_upload = false)
     {
         parent::create();
+        if (!$a_upload) {
```

Comment:
wenn es keine expliziten Aufrufe inkl Parameter zu finden gibt würde ich diesen evtl weglassen, aber es in der PR Beschreibung erwähnen